content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -7,21 +7,28 @@ owner: istio/wg-networking-maintainers
 test: no
 ---
 
-## Configuring network topologies
+## Forwarding external client attributes (IP address, certificate info) to workloads
 
 {{< boilerplate experimental >}}
 
-Istio provides the ability to manage settings like [X-Forwarded-For](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for) (XFF)
-and [X-Forwarded-Client-Cert](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-client-cert)
-(XFCC), which are dependent on how the gateway workloads are deployed. This is currently an in-development feature. For more
-information on `X-Forwarded-For`, see the IETF's [RFC](https://tools.ietf.org/html/rfc7239).
+Many applications require knowing the client IP address (e.g for security such as WAF, auditing, observability with
+tracing and logging, caching) from an originating request to behave properly. While Istio provides an
+[ingress gateway](/docs/tasks/traffic-management/ingress/ingress-control/), given the varieties of architectures (e.g.
+ingress gateway behind a Cloud Load Balancer, many intermediate proxies between a gateway and Istio or direct gateway
+exposure to the internet) reasonable defaults are not able to be shipped that support the proper forwarding of a client
+attributes, such as the client IP address, to the appropriate workloads. Similar to a client IP address, other client
```

Comment:
Rather than have these ", such as x," sections, I'd maybe just say up front that we're talking about client IP and client cert information, and simplify the language later.